src/libs/ReportUtils.ts

```diff
@@ -2909,6 +2909,42 @@ function getDisplayNameForParticipant({
     return shouldUseShortForm ? shortName : longName;
 }
 
+function excludeParticipantsForDisplay(
```

Comment:
Can you please add unit tests for this new method, and also a unit test for the component to make sure the participants get excluded?

Reply:
I added unit tests for the method. I think it's a bit difficult to add unit test for the component as using its output is hidden behind a huge wall of if/else if/else statements and since we don't have any tests for those, it's super tricky to figure out the proper test scenarios for that